src/bridge/MatrixHandler.ts

```diff
@@ -1097,7 +1097,7 @@ export class MatrixHandler {
 
     private async textForReplyEvent(event: MatrixMessageEvent, replyEventId: string, ircRoom: IrcRoom):
     Promise<{formatted: string; reply: string}|null> {
-        const REPLY_REGEX = /> <(.*)>(.*)\n\n(.*)/;
+        const REPLY_REGEX = /> <(.*?)>(.*?)\n\n(.*)/;
```

Comment:
Although this no longer breaks with the telegram bridge, it still doesn't do what the spec says you should do:

> To strip the fallback on the `body`, the client should iterate over each line of the string, removing any lines that start with the fallback prefix ("> ", including the space, without quotes) and stopping when a line is encountered without the prefix. This prefix is known as the "fallback prefix sequence".